cmd/pilot-agent/main.go

```diff
@@ -118,24 +136,18 @@ var (
 )
 
 func init() {
+	proxyCmd.PersistentFlags().StringVar(&config, "config", "",
+		"Configuration overrides for the proxy as YAML/JSON")
 	proxyCmd.PersistentFlags().StringVar((*string)(&serviceregistry), "serviceregistry",
 		string(platform.KubernetesRegistry),
 		fmt.Sprintf("Select the platform for service registry, options are {%s, %s}",
 			string(platform.KubernetesRegistry), string(platform.ConsulRegistry)))
-	proxyCmd.PersistentFlags().StringVar(&meshconfig, "meshconfig", "/etc/istio/config/mesh",
```

Comment:
Backward compat please. Easy to copy old flags to new.

Reply:
meshconfig is being deprecated and into two resources...

Reply:
No problem with deprecating - as long as it's kept around for a while before it is removed. As is common with any other 
project exposing flags or configs or APIs ( and required in most google3 projects).